views/js/e2e/runner/_setup/setupCommands.js

```diff
@@ -92,5 +95,43 @@ Cypress.Commands.add('setDeliveryForGuests', () => {
 
     // Wait until save happened properly
     // Not ideal but these requests have to be waited in this order upon delivery save
-    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery' ]);
-});
\ No newline at end of file
+    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery', '@getData' ]);
+});
+
+Cypress.Commands.add('startTest', () => {
+    // some windows workaround
+    cy.wait(500);
+
+    cy.get(runnerSelectors.testList)
+        .find(runnerSelectors.availableDeliveries)
+        .contains('Delivery of e2e example test')
+        .click();
+
+    cy.wait(['@testRunnerGet', '@testRunnerPost']);
+});
+
+Cypress.Commands.add('nextItem', () => {
+    cy.get(setupSelectors.testNavigation.nextItem).click();
+    cy.wait(['@testRunnerGet', '@testRunnerPost']);
+});
+
+Cypress.Commands.add('previousItem', () => {
+    cy.get(setupSelectors.testNavigation.previousItem).click();
+    cy.wait(['@testRunnerGet', '@testRunnerPost']);
+});
+
+Cypress.Commands.add('skipItem', () => {
+    cy.get(setupSelectors.testNavigation.skipItem).click();
+    cy.wait(['@testRunnerGet', '@testRunnerPost']);
+});
+
+Cypress.Commands.add('endTest', () => {
```

Comment:
endTest and skipAndEnd should be somehow also a cleanup commands

Reply:
Not sure.

Reply:
Maybe not needed because by testing the interactions, tools we dont have to really end the tests just leaving the page.